commons/protocol/src/main/scala/sbt/protocol/Keys.scala

```diff
@@ -98,3 +98,19 @@ object MinimalBuildStructure {
   implicit val unpickler: Unpickler[MinimalBuildStructure] = genUnpickler[MinimalBuildStructure]
   implicit val pickler: Pickler[MinimalBuildStructure] = genPickler[MinimalBuildStructure]
 }
+
+final case class BuildDependencies(
+  classpath: Map[ProjectReference, Seq[ClasspathDep]],
+  aggregate: Map[ProjectReference, Seq[ProjectReference]])
+object BuildDependencies {
+  implicit val unpickler: Unpickler[BuildDependencies] = genUnpickler[BuildDependencies]
+  implicit val pickler: Pickler[BuildDependencies] = genPickler[BuildDependencies]
```

Comment:
These two vals can be one val `implicit val picklerUnpickler = PicklerUnpickler.generate[BuildDependencies]` now; there are a bunch of examples like the above leftover from before we had PicklerUnpickler.